crawl-ref/source/mutation-data.h

```diff
@@ -2119,6 +2119,52 @@ static const mutation_def mut_data[] =
   {"Your legs no longer feel aquatic."},
 },
 
+{ MUT_FLOAT, 0, 1, mutflag::good, false,
+  "float",
+
+  {"You float through the air.", "", ""},
+  {"You feel weightless.", "", ""},
+  {"You feel dragged down by the weight of the world."},
+},
+
+{ MUT_INNATE_CASTER, 0, 1, mutflag::good, false,
+  "innate caster",
+
+  {"You learn spells naturally, not from books.", "", ""},
+  {"You feel mystic power welling inside you.", "", ""},
+  {"You feel a greater respect for book-learning."},
+},
+
+{ MUT_FOOTLESS, 0, 1, mutflag::good, false,
+  "footless",
+
+  {"You have no feet.", "", ""},
+  {"You feel footless.", "", ""},
+  {"You feel footloose."},
```

Comment:
I was vaguely concerned that it might not be obvious to people that dj don't have feet, or why they don't, but agree this isn't worth a whole line on A. I'll merge it.